### PR TITLE
*: add main func to etcd-backup-operator

### PR DESCRIPTION
**Pull request by fanminshi**

### CODE REVIEW

**Comment by fanminshi** [comment]
@hongchaodeng ptal

File: etcd-backup-operator/cmd/backup-operator/main.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+func createRecorder(kubecli kubernetes.Interface, name, namespace string) record.EventRecorder {
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartLogging(logrus.Infof)
+	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(kubecli.Core().RESTClient()).Events(namespace)})
+	return eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: name})
+}
+
+func run(stop <-chan struct{}) {
+	// b := operator.New()
```
Comment (hongchaodeng) [inline etcd-backup-operator/cmd/backup-operator/main.go]: Remove commented code

File: etcd-backup-operator/cmd/backup-operator/main.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+)
+
+func main() {
+	namespace := os.Getenv("MY_POD_NAMESPACE")
```
Comment (hongchaodeng) [inline etcd-backup-operator/cmd/backup-operator/main.go]: Put all these into a func ?

Comment (fanminshi) [reply etcd-backup-operator/cmd/backup-operator/main.go]: `namespace`, `name`,  and `id` are being used by code down below.

Comment (hongchaodeng) [reply etcd-backup-operator/cmd/backup-operator/main.go]: ok

**Comment by fanminshi** [comment]
via https://jenkins-etcd-public.prod.coreos.systems/job/etcd-operator/2369/

```sh
22:27:02 --- FAIL: TestDisasterRecoveryAll (110.44s)
22:27:02 	crd_util.go:45: creating etcd cluster: test-etcd-36ndf
22:27:02 	util.go:82: 2017-09-28 22:23:00.918482644 +0000 UTC m=+17.019641288 waiting size (3), healthy etcd members: names ([])
22:27:02 	util.go:82: 2017-09-28 22:23:10.778099051 +0000 UTC m=+26.879257733 waiting size (3), healthy etcd members: names ([])
22:27:02 	util.go:82: 2017-09-28 22:23:20.778729412 +0000 UTC m=+36.879888080 waiting size (3), healthy etcd members: names ([])
22:27:02 	util.go:82: 2017-09-28 22:23:30.778094129 +0000 UTC m=+46.879252808 waiting size (3), healthy etcd members: names ([test-etcd-36ndf-0000 test-etcd-36ndf-0001])
22:27:02 	util.go:82: 2017-09-28 22:23:40.778628303 +0000 UTC m=+56.879786973 waiting size (3), healthy etcd members: names ([test-etcd-36ndf-0001 test-etcd-36ndf-0002 test-etcd-36ndf-0000])
22:27:02 	recovery_test.go:131: failed to create backup pod: still failing after 6 retries
22:27:02 	crd_util.go:77: deleting etcd cluster: test-etcd-36ndf
22:27:02 	crd_util.go:90: waiting backup deleted of cluster (test-etcd-36ndf)
```

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**